facturacion_mexico/facturacion_fiscal/tests/test_tipo_e_nota_credito.py

```
@@ -0,0 +1,197 @@
+	}
+	si.get = lambda key, default=None: _data.get(key, default)
+	return si
+
+
+# ── Tests guard bloqueante ────────────────────────────────────────────────────
+
+
+class TestTipoEGuard(FrappeTestCase):
+	"""Guard en _prepare_facturapi_data: tipo E sin UUID bloquea timbrado."""
+
+	def _run_tipo_e_block(self, uuid_relacionado: str):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_run_tipo_e_block`

(ANN202)